### PR TITLE
9006 added the attachments to bulk import docs

### PR DESCRIPTION
**Pull request by pradeepmurugesan**
docs for the PR: mattermost/mattermost-server#9124

ticket: mattermost/mattermost-server#9006

### CODE REVIEW

**Comment by amyblais** [comment]
Thank you! Sorry about that, not sure why rebasing didn't work as I don't think anything has been merged to that branch yet.

**Comment by pradeepmurugesan** [comment]
@amyblais  No.. My bad..  I messed up the PR by changing the base :-)

All good now.. Thanks..

File: source/deployment/bulk-loading-data-format.rst [hunk 1]
```diff
@@ -855,6 +863,13 @@ Fields of the Post object
       <td align="center" valign="middle">Yes</td>
       <td align="center" valign="middle">No</td>
     </tr>
+    <tr class="row-odd">
+      <td valign="middle">attachments</td>
+      <td valign="middle">array</td>
+      <td>Attachments associated with the Post. Must be an array of <a href="#fields-of-the-attachment-object">Attachment</a> objects.</td>
```
[the remark this answers is not in the document]
Comment (amyblais) [reply source/deployment/bulk-loading-data-format.rst]: I changed to "File attachments associated with this post." for both.

**Comment by jespino** [comment]
@pradeepmurugesan Would you like to do the changes for use the Attachments Import feature in the sampledata to allow to generate attachments randomly? Basically I would expect something like `./mattermost sampledata --attachments ./attachments-directory` and then (like in the avatars random selection) select one file, and generate the attachment in the sampledata bulk output.

Sampledata, under the hood, it simply generates a temporary bulk file, and import it into the system.

If you don't like the idea, I can implement it, is not a huge deal, but I think you did a very good job with the Attachments import and I think you could do it pretty well in the sampledata too. :)

**Comment by jespino** [comment]
@pradeepmurugesan Another thing that you can do related to this PR is to add the attachments to the `ImportDirectPost` too. Actually now we can import attachments in posts in channels, replies in channels, and replies in direct posts, but we can't import attachments in direct posts.

**Comment by pradeepmurugesan** [comment]
@jespino Hi.. sure.. I can do it. Is there any time constraints on this ? Coz I am in my work week. Can give it a shot this weekend.. 

Is it ok ? Kindly clarify

**Comment by jespino** [comment]
@pradeepmurugesan sure, no rush. Whenever you can will work for us. I'll create a help wanted ticket for this and directly "assign" it to you (tomorrow) :)

**Pull request merged**